config/initializers/omniauth.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'],
```

Comment:
If this needs to be in place, maybe you can use `ENV.fetch` instead, so it fails early on, when you boot the app.